.github/labeler.yml

```diff
@@ -44,7 +44,6 @@ inlong-sdk:
 
 ci:
   - '.github/workflows/**/*'
-  - '.github/actions/**/*'
```

Comment:
@healchow This path will be used in #3676.
![image](https://user-images.githubusercontent.com/49934421/165941663-b49ca209-2d04-472c-a0b5-229a4eec3569.png)